pybossa/importers/csv.py

```diff
@@ -58,91 +58,179 @@ def error():
 
     return value
 
-class BulkTaskCSVImport(BulkTaskImport):
+class ReservedFieldProcessor(object):
```

Comment:
No big deal since they're all defined in one file, but you could make a `FieldProcessor` base class.